packages/atomic/src/pages/index.html

```diff
@@ -201,6 +205,17 @@
               number-of-values="20"
             ></atomic-segmented-facet>
           </atomic-segmented-facet-scrollable>
+          <atomic-layout-section class="flex" section="popovers">
+            <atomic-popover>
+              <atomic-facet field="objecttype" number-of-values="8"></atomic-facet>
```

Comment:
Need to add ids `facet-id="something"` to all these facets & the segmented one since the field are already used for other facets